ibis/backends/flink/__init__.py

```diff
@@ -435,11 +510,17 @@ def drop_view(
             If `False`, an exception is raised if the view does not exist.
         """
         qualified_name = self._fully_qualified_name(name, database, catalog)
-        if not (self._table_env.drop_temporary_view(qualified_name) or force):
```

Comment:
Is there a reason you do everything with SQL, instead of some of the PyFlink functions? Not sure if something you and @chloeh13q discussed/based on how she's been implementing things, or if it's based on some investigation--just curious.

Reply:
I prefer SQL over PyFlink functions because I find the doc for Flink SQL more clear. In my experience, it is much easier to understand what we can do or cannot do with Flink while reading on the SQL doc relative to the PyFlink doc. I have not discussed this with @chloeh13q though.